phys2bids/cli/run.py

```diff
@@ -125,6 +125,11 @@ def _get_parser():
                           type=str,
                           help='full path to store channels plot ',
                           default='')
+    optional.add_argument('-yml', '--participant-yml',
```

Comment:
When we're going to work with #199, -yml might become confusing (since the heuristic will be a yaml file). Could you find a different short argument?

Reply:
What about calling it -ptable or similar?

Reply:
The description data should also go in the same yaml file to make the user's life easier... so I'm not sure `-ptable` should be its name. When we work with #199, will the heuristic be always a yaml? If that's the case, then `-heur` will be free to use.

Reply:
The file will be given through `-heur`, but my concern is that users might get confused by `-yml`.
Anyway, let's keep it like this.